### DISCUSSION TITLE
xDrip can strobe the flashlight for an alert

### DISCUSSION THREAD

**Discussion by Navid200** [NightscoutFoundation/xDrip#2662] [Show and tell]
A new feature has been added to xDrip. It is in the February 6, 2023 release.

When activated, while the phone is being charged, when an alert triggers, the flashlight flashes. You need to place the phone face down. Otherwise, very likely, no light will be visible.

To enable this, after updating to the required release, enable Settings &#8722;> Alarms and Alerts &#8722;> Glucose Alerts Settings &#8722;> Use Camera Light

**Comment by jamorham** [comment]
Transparent cases and phone stands will also likely make the light more visible. It is intended for use at night to increase the chances of an alarm waking the user up.

**Reply by Navid200** [reply]
There is a father whose daughter can only wake up with light and he was looking for using IFTTT in Nightscout.
I have informed him.  This may be useful to them.

Thanks for adding it.
